picard/webservice.py

```diff
@@ -484,30 +486,35 @@ def _find(self, entitytype, handler, kwargs):
             value = QUrl.toPercentEncoding(unicode(value))
             queryargs[str(name)] = value
         path = "/ws/2/%s" % (entitytype)
-        return self.get(host, port, path, handler, queryargs=queryargs)
+        return self.get(host, port, path, handler, queryargs=queryargs,
+                xml=xml, priority=priority, important=important, mblogin=mblogin,
+                cacheloadcontrol=cacheloadcontrol, refresh=refresh)
 
     def find_releases(self, handler, **kwargs):
-        return self._find('release', handler, kwargs)
+        return self._find('release', handler, **kwargs)
 
     def find_tracks(self, handler, **kwargs):
-        return self._find('recording', handler, kwargs)
+        return self._find('recording', handler, **kwargs)
 
     def find_artists(self, handler, **kwargs):
-        return self._find('artist', handler, kwargs)
+        return self._find('artist', handler, **kwargs)
 
-    def _browse(self, entitytype, handler, kwargs, inc=[], priority=False, important=False):
+    def _browse(self, entitytype, handler, inc=[],
+            xml=True, priority=False, important=False, mblogin=False,
```

Comment:
One more thing: why is _browse now neither important nor has it priority? It's used for the other versions context menu item, which seems to be of similar importance as the search for similar tracks...